roles/network-multus/templates/ovs-cni.yml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ ovs_provisioner_name }}
+  namespace: {{namespace}}
+  labels:
+    tier: node
+    app: ovs-cni-plugin
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: ovs-cni-plugin
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name:  {{ ovs_provisioner_name }}
+        image: {{ ovs_provisioner_repo }}:{{ ovs_provisioner_release }}
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+          limits:
+            cpu: "100m"
+            memory: "50Mi"
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - name: cnibin
+          mountPath: /host/opt/cni/bin
+      volumes:
+      - name: cnibin
+        hostPath:
+          path: /opt/cni/bin
```

Comment:
We need to install ovs-vsctl on the node, could you please also install something based on following definition in case we are on openshift, please:

```yaml
---
apiVersion: extensions/v1beta1
kind: DaemonSet
metadata:
  name: ovs-vsctl-amd64
  namespace: kube-system
  labels:
    tier: node
    app: ovs-cni-plugin
spec:
  template:
    metadata:
      labels:
        tier: node
        app: ovs-cni-plugin
    spec:
      nodeSelector:
        beta.kubernetes.io/arch: amd64
      tolerations:
      - key: node-role.kubernetes.io/master
        operator: Exists
        effect: NoSchedule
      containers:
      - name: ovs-vsctl
        command:
        - /bin/bash
        - -c
        - |
          cp /usr/bin/ovs-vsctl /host/usr/bin/_ovs-vsctl
          echo '#!/usr/bin/bash
          _ovs-vsctl --db unix:///run/openvswitch/db.sock $@
          ' > /host/usr/bin/ovs-vsctl
          chmod +x /host/usr/bin/ovs-vsctl
          sleep infinity
        image: docker.io/openshift/origin-node:v3.10.0-rc.0
        resources:
          requests:
            cpu: "100m"
            memory: "50Mi"
          limits:
            cpu: "100m"
            memory: "50Mi"
        securityContext:
          privileged: true
        volumeMounts:
        - name: usrbin
          mountPath: /host/usr/bin
      volumes:
        - name: usrbin
          hostPath:
            path: /usr/bin
```

Reply:
Done

Reply:
AFAIK it breaks openshift-sdn if we install another instance of OVS on the node